lib/init.coffee

```diff
@@ -93,6 +93,11 @@ module.exports =
       default: true
       description: "enable experimental strict Optional checks"
       order: 14
+    followImports:
+      type: 'string'
+      default: 'silent'
+      description: "how to treat imports (normal,silent,skip,error)"
```

Comment:
Should use an array of valid possible values ["normal", "silent", "skip", "error"].  It would prevent the possibility to provide invalid value which would trigger error pop-up.

At this link: https://atom.io/docs/api/v1.17.2/Config
It describes the syntax to do it in the section "Other Supported Keys" -> "enum"